src/test/java/unicash/logic/commands/ClearTransactionsCommandTest.java

```diff
@@ -32,4 +37,40 @@ public void execute_nonEmptyUniCash_success() {
                 ClearTransactionsCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
+    @Test
+    public void multipleClearTransactionsCommand_equalsTrue() {
+        ClearTransactionsCommand clearCommand = new ClearTransactionsCommand();
+        assertEquals(clearCommand, new ClearTransactionsCommand());
+
+    }
+
+    @Test
+    public void sameClearTransactionsCommand_equalsTrue() {
+        ClearTransactionsCommand clearCommand = new ClearTransactionsCommand();
+        assertTrue(clearCommand.equals(clearCommand));
+        assertTrue(clearCommand.equals(new ClearTransactionsCommand()));
+
+    }
+
+    @Test
+    public void equalsMethod_differentCommandTypes_returnsFalse() {
+        Command resetCommand = new ResetCommand();
+        Command clearCommand = new ClearTransactionsCommand();
+        assertNotEquals(resetCommand, clearCommand);
+        assertFalse(clearCommand.equals(resetCommand));
+        assertFalse(clearCommand.equals(new ResetCommand()));
+    }
+
+    @Test
+    public void equalsMethod_nullInput_returnsFalse() {
+        assertNotEquals(null, new ClearTransactionsCommand());
+    }
+
+    @Test
+    public void toStringTest() {
```

Comment:
Also, good to standardize how we wrote our `toString` test name so if we're using `toStringMethod`, best to keep it uniform

Reply:
Thanks, I've changed other instances of this as well